src/components/Mutations/DepositDetailModal.vue

```diff
@@ -13,24 +16,39 @@ defineProps({
 
 <template>
   <div class="flex flex-column">
-    <div class="flex flex-row justify-content-between">
-      <div class="transaction-left-column"><p>{{ $t("transactions.total") }}</p></div>
-      <div class="transaction-right-column"><p>{{ formatPrice(depositInfo.amount) }}</p></div>
-    </div>
-    <div class="flex flex-row justify-content-between">
-      <div class="transaction-left-column"><p>{{ $t("transactions.to") }}</p></div>
-      <div class="transaction-right-column">
-        <p>{{ depositInfo.deposit?.to.firstName + ' ' +depositInfo.deposit?.to.lastName }}</p>
-      </div>
-    </div>
-    <div class="flex flex-row justify-content-between">
-      <div class="transaction-left-column"><p>{{ $t("transactions.transferRef") }}</p></div>
-      <div class="transaction-right-column"><p>{{ depositInfo.deposit?.stripeId }}</p></div>
-    </div>
-    <div class="flex flex-row justify-content-between">
-      <div class="transaction-left-column"><p>{{ $t("transactions.transferType") }}</p></div>
-      <div class="transaction-right-column"><p>{{ $t("transactions.deposit") }}</p></div>
-    </div>
+    <span>
+      {{ new Date(depositInfo.createdAt!!).toLocaleString('nl-NL', {
```

Comment:
Wil je dit niet in een computed hebben? Dat je in je template één variable er maar in hoeft te zetten.